tests/virt_autotest/virt_utils.pm

```diff
@@ -302,4 +302,79 @@ sub upload_supportconfig_log {
     save_screenshot;
 }
 
+# Download guest image and xml from a NFS location to local
+# the image and xml is coming from a guest installation testsuite
+# need set SKIP_GUEST_INSTALL=1 in the test suite settings
+# only available on x86_64
+sub download_guest_assets {
+
+    # guest_pattern is a string, like sles-11-sp4-64, may or may not with pv or fv given.
+    my ($guest_pattern, $vm_xml_dir) = @_;
+
+    # list the guests matched the pattern
+    my $qa_guest_config_file = "/usr/share/qa/virtautolib/data/vm_guest_config_in_vh_update";
+    my $guest_list           = script_output "source /usr/share/qa/virtautolib/lib/virtlib; get_vms_from_config_file $qa_guest_config_file $guest_pattern";
+    my $install_guest_list   = "";
+    if (get_var('SYSTEM_ROLE', '') eq "kvm") {
```

Comment:
PLease refer to my comment in https://github.com/SUSE/qa-automation/pull/690/ for function get_vms_from_config_file.